Roslyn.sln

```diff
@@ -410,6 +410,8 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.CodeAnalysis.Exte
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Net.Compilers.Toolset.Package", "src\NuGet\Microsoft.Net.Compilers.Toolset\Microsoft.Net.Compilers.Toolset.Package.csproj", "{A74C7D2E-92FA-490A-B80A-28BEF56B56FC}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.CodeAnalysis.ExternalAccess.Xamarin.Remote", "src\Tools\ExternalAccess\Xamarin.Remote\Microsoft.CodeAnalysis.ExternalAccess.Xamarin.Remote.csproj", "{6707896F-6F69-4DA0-9DD2-F1C6FB4BB939}"
```

Comment:
Which product is this targeting? I thought we didn't need any external access libraries because the current consumption is a pull model where they're adjusting to any breaks?

Reply:
My understanding is this was Xamarin _in Visual Studio_. Does VS for Mac / MonoDevelop use ServiceHub/OOP?

Reply:
We've not done any work towards supporting OOP, not is there any official plan to work on integrating it yet.